lib/fluent/plugin/base.rb

```diff
@@ -70,6 +76,19 @@ def multi_workers_ready?
         true
       end
 
+      def acquire_worker_lock(name, &block)
+        if fluentd_lockdir.nil?
+          raise RuntimeError, "fail to create lockfile on '#{fluentd_lockdir}'"
+        end
+
+        name = name.gsub(/[^a-zA-Z0-9]/, "_")
+        lockfile = "fluentd-#{name}.lock"
+        File.open(File.join(fluentd_lockdir, lockfile), "w") do |f|
+          f.flock(File::LOCK_EX)
+          block.call()
```

Comment:
```suggestion
          yield
```

Reply:
If using `yield`, then we don't need `$block` argument.

Reply:
Fixed. Thank you.